### DISCUSSION TITLE
When I tried to start on my mac m2 , keep failing

### DISCUSSION THREAD

**Discussion by Guiwoo** [lllyasviel/Fooocus#3013] [Q&A]
![스크린샷 2024-05-27 오전 10 27 24](https://github.com/lllyasviel/Fooocus/assets/77892148/b434af52-c197-43d2-9b99-6a2af4070bdc)

Mac m2 

when i type python entry_with_update.py    

Update failed.
No module named 'pygit2'

Version mismatch for torchsde: Installed version 0.2.6 does not meet requirement torchsde==0.2.5
Installing requirements

**Comment by mashb1t** [comment]
Hey @Guiwoo, make sure you're using Python 3.10 or 3.11 and double check if the packages are installed. You may have to activate the venv first.
Further MacOS installation instructions can be found in the readme.
